helpers/lib_tasks_pytest.py

```diff
@@ -874,6 +874,121 @@ def run_coverage_report(  # type: ignore
         _publish_html_coverage_report_on_s3(aws_profile)
 
 
+def _get_inclusion_settings(target_dir: str) -> Tuple[str, Optional[str]]:
+    """
+    Determine include/omit glob patterns for the coverage report.
```

Comment:
I'd add some examples, since it's not so clear why we need this function.

Reply:
Yes let's explain how it's used (and not only what it does)